src/tabs/borderless.scss

```diff
@@ -4,49 +4,49 @@
 @import './tabs';
 
 @mixin iui-tabs-borderless {
-  @include iui-tabs;
-  margin: $iui-baseline/2 0;
+  .iui-tab {
+    height: $iui-component-height;
```

Comment:
We could just have `padding: 11px 12px` and it looks the same for default size and a bit bigger on large. But then we wouldn't need to set height.

Reply:
Oh, I see why you did this. I saw `None` tab is not fully taking height. But you could apply `display: flex` to the `li` element and it will take full height.

Reply:
Is there any downside to using height?  By using height I can guarantee that the tab height = buttons, text fields, selects height.

Reply:
Maybe it is ok to have controlled height. But then need to think whether we want to hide description when using default size (like in Header breadcrumbs) or it is ok to have default sized Tab with description.

Reply:
It's getting a little too tight to display a secondary line when in the default tab size.
![Screen Shot 2021-06-17 at 12 42 46 PM](https://user-images.githubusercontent.com/849817/122439292-8718d780-cf69-11eb-8ddb-a1e39efcba80.png)

I've changed the CSS so the secondary line only displays when `.iui-large` is applied to the tab group.